pkg/apiserver/apiserver_test.go

```diff
@@ -201,7 +204,7 @@ func handleLinker(storage map[string]rest.Storage, selfLinker runtime.SelfLinker
 }
 
 func newTestRequestInfoResolver() *RequestInfoResolver {
-	return &RequestInfoResolver{sets.NewString("api", "apis"), sets.NewString("api")}
+	return &RequestInfoResolver{sets.NewString("api", "apis"), sets.NewString()}
```

Comment:
> why isn't api considered groupless any more?

For the purposes of this test, it isn't.  Alternatively, I could change the prefix for every test.